.env.rds.example

```
@@ -0,0 +1,42 @@
+# AWS RDS PostgreSQL Configuration
+# Copy this file to .env.local and update with your RDS credentials
+
+# RDS Connection Settings
+RDS_HOST=your-rds-instance.region.rds.amazonaws.com
+RDS_PORT=5432
+RDS_DATABASE=hallucifix
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RDS_DATABASE key should go before the RDS_HOST key

(UnorderedKey)